.pylintrc

```diff
@@ -315,7 +315,7 @@ indent-after-paren=4
 indent-string='    '
 
 # Maximum number of characters on a single line.
-max-line-length=100
+max-line-length=120
```

Comment:
Is that really necessary? Pep suggests maximum line length of 80

Reply:
Our prospector.yaml has pep8 set to check for 120 and the .pylintrc had 100 (which lead to no pep8 error but pylint errors). I'm just making it consistent and 120 fits on my screen pretty much perfectly. 100 leads to a lot of unnecessary linebreaks when importing etc.